src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -81,21 +90,27 @@ public function __construct(Parser $phpParser)
         $this->nodeTraverser = new NodeTraverser();
         $this->nodeTraverser->addVisitor(new NameResolver());
         $this->nodeTraverser->addVisitor($this->cachingVisitor);
+
+        self::$classMap    = array_change_key_case(PhpStormStubsMap::CLASSES);
+        self::$functionMap = array_change_key_case(PhpStormStubsMap::FUNCTIONS);
+        self::$constantMap = array_change_key_case(PhpStormStubsMap::CONSTANTS);
```

Comment:
I was looking at this and being confused: what is the casing?

PHP's default values are annoying...